src/main/java/net/mrscauthd/beyond_earth/events/Events.java

```diff
@@ -217,17 +213,15 @@ public static void itemRender(RenderHandItemEvent.Pre event) {
                 event.setCanceled(true);
             }
 
-            Item item1 = player.getMainHandItem().getItem();
-            Item item2 = player.getOffhandItem().getItem();
-
-            if (item1 instanceof VehicleItem) {
-                if (event.getHandSide() == HumanoidArm.LEFT) {
+            if (event.getHandSide() == HumanoidArm.LEFT) {
+                Item mainHandItem = player.getMainHandItem().getItem();
+                if (mainHandItem instanceof VehicleItem) {
```

Comment:
The variable item1 is now called mainHandItem, item2 is now called offHandItem.

Only getting the mainHandItem if getHandSide() is left.